WooCommerce/Classes/ViewRelated/ReusableViews/TableFooterView.swift

```diff
@@ -55,25 +57,43 @@ extension TableFooterView {
         }
     }
 
-    /// Footnote label text.
+    /// Initialization method for footnote textview
     ///
-    var footnoteText: String? {
-        get {
-            return footnote?.text
-        }
-        set {
-            footnote?.text = newValue
-        }
+    func setupFootnoteLabel() {
+        footnote.attributedText = hiringAttributedText
+        footnote.adjustsFontForContentSizeCategory = true
+        footnote.textContainerInset = .zero
+        footnote.textAlignment = .center
+        footnote.linkTextAttributes = [
+            .foregroundColor: StyleManager.wooCommerceBrandColor,
+            .underlineColor: StyleManager.wooCommerceBrandColor,
+            .underlineStyle: NSUnderlineStyle.single.rawValue
+        ]
     }
+}
 
-    /// Footnote label color.
+
+// MARK - Private methods
+//
+private extension TableFooterView {
+
+    /// Returns the Settings Footer Attributed Text
+    /// (which contains a link to the "Work with us" URL)
     ///
-    var footnoteColor: UIColor? {
-        get {
-            return footnote?.textColor
-        }
-        set {
-            footnote?.textColor = newValue
-        }
+    var hiringAttributedText: NSAttributedString {
```

Comment:
This table footer view wasn't meant to be a reusable view in the first place, hence the attributed string text living in the footer view's file. Should I move the .xib file and code to the Dashboard > Settings folder?

Reply:
How likely do you think it is that we are going to reuse it?

Reply:
Currently `TableFooterView` is used in both `AboutViewController` and `SettingsViewController`, I think it'd be nice to keep it reusable as it is until the design calls for more table footer view layout variations.

One way to keep it reusable is to move the footnote text view setup from `TableFooterView` to its consumer (`SettingsViewController` and `AboutViewController`). Because the `footnote: UITextView` is already public, the footnote setup including `var hiringAttributedText: NSAttributedString` and `func setupFootnoteLabel()` could be moved to the view controller. What do you think?